source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,139 @@
+
+OVA is a known format that is used by different organizations and virtualization platforms. Its specification can be found [here](http://www.dmtf.org/standards/ovf). The standard aims at being an ["open, secure, portable, efficient and extensible format for the packaging and distribution of software to be run in virtual machines"](https://en.wikipedia.org/wiki/Open_Virtualization_Format).  
+
+Recently, oVirt was enhanced with support for importing OVAs that were generated by VMware. In addition, oVirt makes an extensive use of the OVF standard to describe VM configuration in, e.g., VM snapshots and exported VMs.  
+
+The ability to package all the VM data, optionally compressed, in one file eases the processes of backing up a VM and porting a VM to a different platform or hypervisor. Unfortunately, the portability is limited by compatibility issues. Different disk formats, drivers, guest-agents, and OVF structure, hinder the ability to import any OVA to any platform. Therefore, different organizations develop their own conversions tools, like Red Hat's [virt-v2v](http://libguestfs.org/virt-v2v.1.html), for converting VMs from other platforms to their platform.   
+
+# Scope
+This feature is about enhancing the import VM and export VM processes in oVirt by making a broader use of the OVA format.  
+
+## Upload OVA
+Today, the process of importing an OVA is limited to OVA files compatible with those produced by VMware and requires one to manually copy the OVA file into one of hosts managed by oVirt and set the file permissions to vdsm:kvm. It is a cumbersome and error-prone process.  
+
+This feature provides a broader and more user-friendly alternative that enables one to upload a local OVA file straight from the browser (much like the upload-image functionality that was recently added to oVirt).  
+
+It would be possible to upload OVA files that were generated either by oVirt or by external sources. OVA files that were generated by external sources will be converted using virt-v2v, thus the supported external sources are limited to these whose OVA files can be converted by virt-v2v.  
+
+## Export as OVA
+Today, users are provided with two ways to migrate VMs and templates between environments. First, one can use an export domain. However, this approach is suitable only for migration between different oVirt environments. Moreover, there is a plan to drop the support for export domains. Second, one can detach a data domain and attach it to another setup. This way is tailored to disaster recovery and less fits the migration flow due to the side effects in detaching a data domain.  
+
+This feature provides an alternative way for a migration between environments by generating an OVA from an existing VM or template. By encapsulating all VM/template data (optionally compressed) in a single file, we achieve an improved portability. By being aligned with the common OVA specification (rather than our unique structure of a storage domain), we achieve 
```

Comment:
I wouldn't focus on the "compress" at OVA level. I do not think it makes much sense since we have qcow compression for oVirt<->oVirt. Similarly for VMware I haven't seen too many compressed disks in practice